plugins/playlist.html

```diff
@@ -6,20 +6,9 @@
 
   <p>Adds a playlist capability to wavesurfer.</p>
 
-  <h2>Parameters</h2>
-  <p>
-    The `playlistFile` parameter requires a valid http link to a supported playlist file type.<br/><br/>
-    The `playlistType` parameter will default to `m3u` unless specified.<br/>
```

Comment:
@mspae yea I removed that section, I already added the tables.

btw I added the version badge to the footer file and linked to github project page, what do you think?